wger/settings.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
Kindly remove this settings file. We already have another settings file configured to use environmental variables.